charts/trustify/templates/services/importer/030-StatefulSet.yaml

```diff
@@ -57,17 +59,9 @@ spec:
             {{- include "trustification.application.extraVolumeMounts" $mod | nindent 12 }}
 
       volumes:
+        - name: workdir
```

Comment:
`emptyDir` might drain the resources of a node. Leading to the termination of the pod. Therefore, we should use a PVC, even an emphemeral one: https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes

Reply:
done